tests/testthat/test-accuracy-published.R

```diff
@@ -0,0 +1,245 @@
+
+
+test_that(".OneParExpNormalPrior works as expected", {
```

Comment:
```suggestion
test_that("OneParExpNormalPrior works as expected", {
```
also maybe modify the description to something more specific, e.g. " ... reproduces same numbers as in paper" or so?

Reply:
done